crates/matrix-sdk-base/src/latest_event.rs

```diff
@@ -391,7 +397,7 @@ mod tests {
     }
 
     #[test]
-    fn test_different_types_of_messagelike_are_unsuitable() {
+    fn test_stickers_are_suitable() {
```

Comment:
Thanks; can you add another test for an unsuitable messagelike event?

Reply:
done!